keras/layers/merge.py

```diff
@@ -18,6 +18,31 @@ def __init__(self, **kwargs):
     def _merge_function(self, inputs):
         raise NotImplementedError
 
+    def _compute_elemwise_op_output_shape(self, shape1, shape2):
+        """check if shapes of 2 tensors are compatible for element wise operations,
```

Comment:
Format the docstring in the style of other docstrings in the codebase. In particular, needs `# Arguments`, `# Returns` and `# Raises`